bokehjs/src/less/examiner.less

```diff
@@ -137,16 +141,20 @@
   border-width: 1px;
   border-style: solid;
   border-radius: 4px;
-  color: #202124;
-  background-color: #f1f3f4;
-  border-color: #cacdd1;
+  color: var(--color);
+  // @mattpap - I couldn't find a good substitute for this color from the colors
+  // already defined in base.less. We might need something like "muted
+  // background color"
+  background-color: var(--muted-bg-color);
+  border-color: var(--border-color);
 }
 
 .expander {
   margin: 0 2px;
   display: inline-block;
   vertical-align: middle;
-  background-color: #5f6368;
+  // @mattpap - same here, we might need something like "muted color"
+  background-color: var(--muted-color);
```

Comment:
That should probably be just icon color. Looking at the following lines, this may predate SVG images, so I will definitely want this to get migrated to use icons, so everything else, including relevant colors, will be inherited.

Reply:
That creates low contrast with the background (2.5:1), so I'd rather not change it to match --icon-color.